src/examples/mfem_common_plane.cpp

```diff
@@ -344,14 +350,21 @@ int main( int argc, char** argv )
   // for computing the maximum common plane timestep and, if activated, gap rate
   // penalty.
   tribol::registerMfemVelocity(coupling_scheme_id, velocity);
-  // The type of penalty enforcement is set here (i.e. rate vs. kinematic and
-  // the method of setting the penalty value).
-  tribol::setPenaltyOptions(
```

Comment:
Is this no longer needed? Is this call made elsewhere?

Reply:
It's done in `setMfemKinematicConstantPenalty()`